### DISCUSSION TITLE
used zodios for an API client generator (from OpenAPI)

### DISCUSSION THREAD

**Discussion by astahmer** [ecyrbe/zodios#103] [General]
Hey, just wanted to thank you for this library and let you know that I used it to generate [a complete zodios client](https://github.com/astahmer/openapi-zod-client) using OpenAPI specs as input

**Comment by ecyrbe** [comment]
This is really nice.  
i'll add an ecosystem paragraph to mention this.  
  
We will soon be full circle as i'm also making an express implementation using zodios api declaration, that will generate open api json :)
A little like tRPC. But with full validation (tRPC client does not validate).  
I'm taking my time to implement this in my spare time.   
I'm expecting to have something testable in one/two weeks or so.

**Reply by spost-stateless** [reply]
Hey @ecyrbe, how is this effort going?  OpenAPI generation would be super useful for us and we'd love to contribute to that work if there's anything we can do to help.

**Reply by ecyrbe** [reply]
@spost-stateless `zodios/openapi` package is out.
You can check the docs here : http://www.zodios.org/docs/api/openapi

**Comment by ecyrbe** [comment]
Hello, i plan to add `@zodios/openapi` this week. 
Next on my roadmap:
1. help Alex on Api client generator
2. Finish `@zodios/fastify` implementation
3. New `@zodios/vue` package for vue-query
